src/rules/no_mixed_spaces_and_tabs.rs

```diff
@@ -89,6 +89,33 @@ impl LintRule for NoMixedSpacesAndTabs {
       }
     }
   }
+
+  fn docs(&self) -> &'static str {
+    r#"Disallows mixed spaces and tabs for indentation.
+
+Most code conventions require either tabs or spaces be used for indentation. Therefore, if a line of code is indented with both tabs and spaces, it's most likely a mistake of a developer.
+
+### Invalid:
+
+```typescript
+function add(x: number, y: number) {
+    return x + y; // indented with a tab + two spaces
+}
+```
+
+```typescript
+    let x = 5, // indented with a tab + two spaces
+        y = 7;
+```
```

Comment:
Still seems like these examples consist of only spaces, right? Would be good to actually have a tab character and a space character.